src/displayBackend/wayland/CMakeLists.txt

```diff
@@ -53,4 +53,4 @@ if(WITH_IVI_EXTENSION)
 	)
 endif()
 
-target_link_libraries(display_wayland xenbe)
\ No newline at end of file
+target_link_libraries(display_wayland xenbe display_common)
```

Comment:
No newline at the end of file?

Reply:
Well, there was no previously new line, so I didn't add one